readme.md

```diff
@@ -26,6 +32,14 @@ To install Administrator as a Composer package to be used with Laravel 4, simply
 )
 ```
 
+### Laravel 5
+
+Then publish the config file with `php artisan vendor:publish`. This will add the file `app/config/administrator.php`. This [config file](http://administrator.frozennode.com/docs/configuration) is the primary way you interact with Administrator.
```

Comment:
I believe `/config` is now in the root of the project, not in the `/app` directory.